cli.js

```diff
@@ -29,32 +34,66 @@ var usage = `Usage
 Work in progress! Learn more at github.com/cabal-club
 `
 
-if (args.key) {
-  args.key = args.key.replace('cabal://', '').replace('cbl://', '').replace('dat://', '').replace(/\//g, '')
-  args.db = rootdir + args.key
+// Attempt to load local or homedir config file
+try {
+  var config
+  var configFilename = '.cabal.yml'
```

Comment:
I agree that we should add that!

At the moment it will attempt to load a config file first from the current directory. Does that seem like a good thing to do also?

If so, then perhaps the order of finding a config file goes:

1. config from --config path
1. config at current dir
1. config in home dir (or perhaps in ~/.cabal dir)

??